src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write your code here
+const elements = [...document.querySelectorAll('span.population')];
```

Comment:
This line of code queries the DOM for all the 'span.population' elements. Querying the DOM is a costly operation. It's better to do this once and store the result in a variable for future use. However, you are doing it right here, but if you were to use the 'elements' variable in another place in your code, you would have to query the DOM again. So, keep in mind to avoid querying the DOM unnecessarily.